README.md

```diff
@@ -383,6 +383,16 @@ Enables or disables the network widget. This widget can be used to set the netwo
 
 Enables or disables the phone widget. This widget can be used to send SMS or phone call the Android virtual device.
 
+### `Baseband`
```

Comment:
I see 2 baseband sections in this file, one is `Baseband` and one is `baseband`.  They have different descriptions, and different SaaS/PaaS compatibility. Is this correct?

Reply:
baseband section was duplicated!
I removed the second section and update the first one